scripts/EN/Cybil

```diff
@@ -3367,49 +3367,49 @@ CleanScreen(); // 0x45ae: 00000064
 DrawBackground(); // 0x45e2: 00000065
-ShowText((7187)"When Cybil finally reaches a door... A man is in the center of the room... An unfamiliar girl is passing something to Harry... The scene fills Cybil's eyes."); // 0x45e6: 0000100c
+ShowText((7187)"When Cybil finally reaches a door…¥nW1A man is in the center of the room…¥nW1An unfamiliar girl is passing something to Harry…¥nThe scene fills Cybil's eyes."); // 0x45e6: 0000100c
 HandleInput(); // 0x45ec: 00000010
-ShowText((7188)"Harry holds something dearly, at his chest... It is a baby. Harry holds it as if it is his own... Or maybe he is thinking of it as Cheryl... He smiles a bit."); // 0x45f0: 0000100c
+ShowText((7188)"Harry holds something dearly, at his chest…¥nIt is a baby.¥nW2Harry holds it as if it is his own…¥nW2Or maybe he is thinking of it as Cheryl…¥nW2He smiles a bit."); // 0x45f0: 0000100c
 HandleInput(); // 0x45f6: 00000010
-ShowText((7189)"¥"It's a mistake...¥" Perhaps Cybil was the only one to realize this. ¥"He must not accept that baby... It will go on forever... With the spinning of the world and time..."); // 0x45fa: 0000100c
+ShowText((7189)"  ¥"It's a mistake…¥"¥nW2Perhaps Cybil was the only one to realize this?¥n  W2¥"He must not accept that baby…¥n  W2It will go on forever…¥n  W1With the spinning of the world and time…"); // 0x45fa: 0000100c
 HandleInput(); // 0x4600: 00000010
-ShowText((7190)"The baby represents the beginning. It is a devil. Look into its eyes and you can tell.¥""); // 0x4604: 0000100c
+ShowText((7190)"  The baby represents the beginning.¥n  W2It is a devil.¥n  W2Look into its eyes and you can tell.¥""); // 0x4604: 0000100c
 HandleInput(); // 0x460a: 00000010
-ShowText((7191)"Summoning the last of her strength Cybil stands. She must tell him. She must tell Harry of his mistake."); // 0x460e: 0000100c
+ShowText((7191)"Summoning the last of her strength W1Cybil stands.¥nW2She must tell him.¥nW2She must tell Harry of his mistake."); // 0x460e: 0000100c
 HandleInput(); // 0x4614: 00000010
-ShowText((7192)"One of Cybil's arms has already caught fire, and indigo smoke is rising from it. ¥"Harry...¥""); // 0x4618: 0000100c
+ShowText((7192)"One of Cybil's arms has already caught fire, W1and indigo smoke is rising from it.¥n¥nW2¥"Harry…¥""); // 0x4618: 0000100c
 HandleInput(); // 0x461e: 00000010
-ShowText((7193)"Cybil staggers towards Harry. Harry is just beyond her extended left arm."); // 0x4622: 0000100c
+ShowText((7193)"Cybil staggers towards Harry.¥nW2Harry is just beyond her extended left arm."); // 0x4622: 0000100c
 HandleInput(); // 0x4628: 00000010
-ShowText((7194)"¥"Look closely at its eyes... Harry... At the baby's eyes...¥" Is what she had planned to say, but all that came from her mouth were unintelligible shrieks."); // 0x462c: 0000100c
+ShowText((7194)"¥"Look closely at its eyes…¥nW2Harry…¥nW2At the baby's eyes…¥"¥n¥nW2Is what she had planned to say, W1but all that came from her mouth was unintelligible wailing."); // 0x462c: 0000100c
```

Comment:
Changed shrieks to wailing to match the correct Japanese. (The Japanese is literally sobbing, but sobbing doesn't come from one's mouth...)